azurerm/resource_arm_logic_app_workflow.go

```diff
@@ -0,0 +1,469 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/logic/mgmt/2016-06-01/logic"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmLogicAppWorkflow() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLogicAppWorkflowCreateUpdate,
+		Read:   resourceArmLogicAppWorkflowRead,
+		Update: resourceArmLogicAppWorkflowCreateUpdate,
+		Delete: resourceArmLogicAppWorkflowDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": locationSchema(),
+
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"parameters": {
```

Comment:
There are two different parameters here. One is the workflow parameter, the other is the definition parameter.